RecoLocalCalo/HGCalRecProducers/interface/HGCalCLUEAlgo.h

```diff
@@ -112,7 +112,11 @@ class HGCalCLUEAlgo : public HGCalClusteringAlgoBase {
     descNestedNoises.add<std::vector<double> >("values", {});
     iDesc.add<edm::ParameterSetDescription>("noises", descNestedNoises);
     edm::ParameterSetDescription descNestedNoiseMIP;
-    descNestedNoiseMIP.add<double>("value", 0 );
+    descNestedNoiseMIP.add<bool>("scaleByDose", false );
+    iDesc.add<edm::ParameterSetDescription>("scaleByDose", descNestedNoiseMIP);
+    descNestedNoiseMIP.add<std::string>("doseMap", "" );
+    iDesc.add<edm::ParameterSetDescription>("doseMap", descNestedNoiseMIP);
+    descNestedNoiseMIP.add<double>("noise_MIP", 1./100. );
```

Comment:
this is already inside the ```noiseMip = cms.PSet``` then there is also a ```_MIP``` in the parameters inside the PSet.
This is a bit too verbose/repetitive.
I see that this is forced by the SimCalorimetry/HGCalSimProducers/python/hgcalDigitizer_cfi content.
So, perhaps OK, even though these long-range copy-pastes are not ideal.